kfs-cam/src/main/java/org/kuali/kfs/module/cam/businessobject/AssetBase.java

```diff
@@ -2170,4 +2174,26 @@ public String getCamsComplexMaintenanceDocumentLookup() {
         return managedLists;
     }
 
+    public List<Note> getBoNotes() {

```

Comment:
Doing a reference check on this in my IDE returns no results, can you help tell me where this is being used?